roles/infrastructure/defaults/main.yml

```diff
@@ -57,10 +57,17 @@ infrastructure_image_plan:
   sku: "9-lvm-gen2"
   version: latest
 
+# infrastructure_image_plan:
+#   name: aap-installer-1701371407.vhd
+#   resource_group: luiz-group
+
 infrastructure_keypair_name: id_rsa
 infrastructure_ssh_public_keys:
   - path: "/home/azureuser/.ssh/authorized_keys"
     key_data: "{{ lookup('file', lookup('env', 'HOME') + '/.ssh/' + infrastructure_keypair_name + '.pub') }}"
 
 infrastructure_managed_disk_type: Premium_LRS
 infrastructure_os_disk_size_gb: 100
+
+aap_installer_inventory_path: "/home/azureuser/inventory"
+aap_admin_password: "abcde"
```

Comment:
```suggestion

```
All of the other required variables were left out so they could be checked with the assert at the beginning of the role.